src/filter.spec.js

```diff
@@ -379,4 +380,16 @@ describe("filter", () => {
       });
   });
 
+  it("can reuse the predicate on backlink-dependencies and filter them", () => {
+    return Promise
+      .resolve(cyclicTablesWithDependencyLinkFixture)
+      .then(filter({
+        filterBacklinks: true,
+        path: ["tableA"],
+        predicate: v => v.identifier === "A3" || v.identifier === "A4"
+      }))
+      .then(data => {
+        expect(data).to.eql(filterFixtureA3AndA4);
```

Comment:
Should we add another test with row `id` and over three layers/tables?

Reply:
What about a test with two chained filters (e.g. modelYear <= 2018 and than salesRegion === USA)? Should be possible, right?

Reply:
There are tests for multiple layers links and also a test for multiple chained filters